_includes/app/txn-sample.py

```diff
@@ -1,20 +1,20 @@
 # Import the driver.
 import psycopg2
+import psycopg2.errorcodes
 
 # Connect to the cluster.
 conn = psycopg2.connect(database='bank', user='maxroach', host='localhost', port=26257)
 
-# Ensure we are running with transactions.
-conn.set_session(autocommit=False)
 
 def onestmt(conn, sql):
     with conn.cursor() as cur:
         cur.execute(sql)
 
+
 # Wrapper for a transaction.
 # This automatically re-calls "op" with the open transaction as an argument
 # as long as the database server asks for the transaction to be retried.
-def txnWrapper(conn, op):
+def run_transaction(conn, op):
```

Comment:
We could add this version of `run_transaction` to the `cockroachdb` python package. The problem is that transaction wrappers like this are kind of framework-specific, and I don't want to confuse people looking for the sqlalchemy run_transaction function by putting a different and subtly incompatible version in another part of the package. I don't think that many people will actually use the raw `psycopg2` interface; they're more likely to use something higher-level. I'm not sure whether it's better here to show the lowest common denominator or if it would be better to rewrite this sample to use sqlalchemy (which is a popular option, but showing it here might alienate fans of other systems). Let's see how the blog post is received first.